DESCRIPTION

```diff
@@ -1,15 +1,3 @@
-BinaryFiles:

```

Comment:
I had added that here to try to remove the warnings but this did not do anything